rakefile.rb

```diff
@@ -182,7 +183,7 @@ def examples
           sh "cross build --release --bin #{name} --target #{target_arch}"
           cp "target/#{target_arch}/release/#{name}", target_dir
         end
-        create_arch_package(target_arch, target_dir, dir, registry, package_config)
+        create_arch_package_sextant(dir, registry, target_arch)
```

Comment:
the rake task is now broken. Until we fully commit to get rid of rake, this should stay the entry point to using sextant